client/components/Login/ConfirmableLogoutMenuItem.jsx

```diff
@@ -27,17 +28,20 @@ class ConfirmableLogoutMenuItem extends Component {
     render() {
         const { open } = this.state;
         const { onCancel, onClick, onConfirm } = this;
+        const { username } = Session.isSessionActive()
+            ? Session.getSession()
+            : {};
 
         return (
             <React.Fragment>
                 <Menu.Item
                     className="confirmablelogoutmenuitem__anchor-style"
-                    content="Log out"
+                    content={`Log out (${username})`}
```

Comment:
Really nice touch adding the username 👌

Reply:
I was getting confused about which user I was 🤣